corehq/toggles/__init__.py

```diff
@@ -2472,3 +2465,12 @@ def domain_has_privilege_from_toggle(privilege_slug, domain):
     TAG_SOLUTIONS_CONDITIONAL,
     [NAMESPACE_DOMAIN]
 )
+
+MOBILE_USER_DEMO_MODE = FrozenPrivilegeToggle(
```

Comment:
How is this actually used now, since the toggle is moved under the PRACTICE_MOBILE_WORKERS privilege?

Reply:
Oh, wait, nevermind, I click that you're doing this to associate the privilege with the toggle.